templates/zerver/keyboard_shortcuts.html

```diff
@@ -65,10 +65,14 @@
           <td class="definition">{% trans %}New private message{% endtrans %}</td>
         </tr>
         <tr>
-          <td class="hotkey">Tab then Enter</td>
+          <td class="hotkey"><p>Tab then Enter,</p><p>Ctrl + Enter</p></td>
```

Comment:
The `<p>` tags make this look really tall; I removed them and just used a `<br />` between the lines